components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -327,24 +324,20 @@ open class GleanInternalAPI internal constructor () {
 
 object Glean : GleanInternalAPI() {
     /**
-    * Enumeration of different metric lifetimes.
-    */
+     * Enumeration of events that might trigger pings to be sent.
```

Comment:
Are we expecting additional events to trigger pings?  If there aren't any known use-cases for other events, I'd be for refactoring this enum away into a constant.  An enum with a single value seems like a weird construct to me.

Reply:
Good point. I refactored that part away! \o/